wiki/Article_Styling_Criteria/en.md

```diff
@@ -304,19 +335,21 @@ Optionally, add an extra line before and after the heading to help make the head
 
 #### Titles
 
-*Notice: The title of an article must never be escaped, as titles are parsed as raw text, not Markdown.*
+*Notice: Article titles must not be escaped, as they are parsed as raw text, not Markdown.*
+
+All articles must have one level 1 heading, being the article's title. Article titles must use title case and never contain formatting, links, nor images. The level 1 heading must be placed at the start of the article unless reference links or metadata placed there. If this is the case, place the level 1 heading underneath it.
 
-All articles must have one level 1 heading, being the article's title. Article titles must use title case and never contain formatting, links, nor images. The title of an article is the name of the folder that the article is located in.
+The title of an article must match the folder name it is located in (spaces can replace underscores (`_`) where appropriate). The exception to this rule are tournament pages, which may use abbreviations, acronyms, or initialisms.
 
-The level 1 heading must be placed at the start of the article unless you have reference links or metadata placed there. If this is the case, place the level 1 heading underneath it.
+---
 
-For English article titles, if you need to reword the title, you must rename the folder to match the article title. There are rare exceptions to this however.
+For English article titles, if the article's title needs to be renamed, the folder name it is in must also be renamed to match it.
 
 #### Sections
 
-All section headings must use levels 2 to 5, never level 6. Section headings must use sentence case and should never contain styles nor links. In addition to this, section headings must never skip a heading level (e.g. do not go from a level 2 heading to a level 4 heading). The first section heading must be a level 2 heading. Unlike titles, section headings can have small image icons.
+All section headings must use levels 2 to 5, never level 6. Section headings must use sentence case and must not contain formatting nor links. In addition to this, section headings must not skip a heading level (e.g. do not go from a level 2 heading to a level 4 heading). The section heading after the title heading must be a level 2 heading. Unlike titles, section headings can have small image icons.
 
-In the osu! wiki, heading levels 4 and above do not appear in the table of contents and are not given the id attribute (this means you cannot link directly to these headings).
+In the osu! wiki, heading levels 4 to 6 do not appear in the table of contents and are not given the id attribute (these cannot be linked directly).
```

Comment:
> All section headings must use levels 2 to 5.
```suggestion
In the osu! wiki, heading levels 4 and 5 do not appear in the table of contents and are not given the id attribute (these cannot be linked directly).
```

Reply:
I thought it should still be mentioned even if it is not to be used, but I suppose it might lead to some confusion.